skyportal/utils/offset.py

```diff
@@ -50,6 +50,21 @@
     f':{cfg["server.port"]}' if cfg['server.port'] not in [80, 443] else ''
 )
 
+NGPS_TARGET_BANDS_TO_SNCOSMO = {
+    'G': ['ztfg', 'sdssg', 'lsstg'],
+    'R': ['ztfr', 'sdssr', 'bessellr', 'standard::r', 'lsstr'],
+    'I': ['ztfi', 'sdssi', 'besselli', 'standard::i', 'lssti'],
+    'U': ['sdssu', 'bessellux', 'standard::u', 'lsstu'],
+}
+SNCOSMO_BANDS_TO_NGPS_TARGET = {}
```

Comment:
add a comment that you are inverting this dictionary?

Reply:
done